src/styles/App.css

```diff
@@ -0,0 +1,77 @@
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  border-bottom: 1px solid #e8e8e8;
```

Comment:
- [ ] Please use CSS variables to save and reuse repeated colors like the e8e8e8 among others that you have repeated.